source/documentation/how-we-handle-security.html.md.erb

```diff
@@ -0,0 +1,62 @@
+---
+title: How does MOJ Forms handle security?
+last_reviewed_on: 2023-08-17
+review_in: 6 months
+---
+
+<% if current_page.url != "/" %># <%= current_page.data.title %><% end %>
+
+## Context
+What is MOJ Forms - Publish digital forms quickly and easily. Give your users a secure, accessible way to contact you, make a request or submit information.
+
+The whole platform is made from separate services that perform different tasks to collect user data in a structured, secure manner.
+
+MOJ Forms is listed as a Data Asset by the MOJ DPIA team, which means the MOJ Forms part of the DPIA process is already assured and only needs referencing. 
+
+## Editor
+The Editor is a staff facing application which provides the tools to build a form which meets the user journey. Add settings and publish to a test environment and then publish to the live environment.
+
+The Form uses json to store the content and branching instructions. This metadata can be stored in the open as it does not contain anything confidential or personal.
+
+Accessing the Editor we use Single Sign On which confirms the authentication with our domains Active Directories.
+
+##Platform
+The main services which lets MOJ Forms work are:
+
+1. Runner - The application is the form, it is citizen facing.
+2. User Datastore - Saves data entered in the form. 
+3. User Filestore - Save upload documents
+4. Malware Scanner - Scans uploaded documents for malware
+5. Submitter - Passes the data to the backend team as directed in the settings for the form.
+
+### Service Token Cache
+This service is used to confirm each application is communicating with the application within the platform. This fulfils one of the principles to make sure we do not reply to the hosting for security between the services. For example, the runner and the user datastore will request a token and only if they match will they communicate.
+
+More information can be found in the [platforms](/#moj-forms-platform) section.
+
+### End User data
+The data is transported over https and encrypted using the session cookie generated when the user starts the form so it is stored encrypted per-user + per-form. This Sequential data is encrypted at rest in an RDS hosted in AWS.
+Documents are checked on upload that they meet the type and size validation and streamed to a malware checker, only then it is stored encrypted in a S3 bucket. Only the internal services can add and get files from the filestore. 
```

Comment:
Might mention that bucket connections are https and bucket contents are encrypted at rest specifically?